client/src/Components/Pages/Questionnaire/Wizard.js

```diff
@@ -1,13 +1,19 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-this-in-sfc */
+/* eslint-disable react/destructuring-assignment */
 import React from 'react';
+
 import './style.css';
 
+
 export const Wizard = ({ step: currentIndex, ...props }) => {
   const steps = React.Children.toArray(props.children);
   const prevStep = currentIndex !== 0 && steps[currentIndex - 1].props;
   const nextStep = currentIndex !== steps.length - 1 && steps[currentIndex + 1].props;
 
+
   const SubmitQhandler = () => {
-     window.location=('/dash');
+    props.hanleUpdate();
```

Comment:
should rename this "handleUpdate" no?

Reply:
what do you mean ?!

Reply:
it's a typo - you spell it "handle" not "hanle". never mind!